server/command/install.go

```diff
@@ -74,7 +74,7 @@ func (s *service) executeInstallHTTP(params *commandParams) (*model.CommandRespo
 
 	m, err := apps.ManifestFromJSON(data)
 	if err != nil {
-		return errorOut(params, err)
+		return errorOut(params, errors.Wrap(err, "unable to read `manifest json` from "+manifestURL))
```

Comment:
`"unable to decode " + manifestURL` seems more intention revealing and succinct?

Reply:
My intention was to reveal to the user that an expected `manifest json` structure was expected, but agree it is possibly over descriptive.  Updated to your suggestion.